File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/Robot.java
package org.firstinspires.ftc.teamcode.robot;

import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;


public class Robot {

    public WheelBase wheelBase;
    public Manipulator manipulator;
    public Lift lift;

    public Hang hang;

    public Camera camera;
    public LinearOpMode linearOpMode;

    public Robot(LinearOpMode linearOpMode1) {
        linearOpMode = linearOpMode1;
        wheelBase = new WheelBase(this);
        lift = new Lift(this);
        manipulator = new Manipulator(this);
        camera = new Camera(this);
        hang = new Hang(this);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/Robot.kt
package org.firstinspires.ftc.teamcode.robot

import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode

class Robot(var linearOpMode: LinearOpMode) {
    @JvmField
    var camera: Camera = Camera(this)

    @JvmField
    var wheelBase: WheelBase = WheelBase(this)

    @JvmField
    var manipulator: Manipulator = Manipulator(this)

    @JvmField
    var lift: Lift = Lift(this)

}